server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/cics/CICSParser.g4

```diff
@@ -451,8 +425,8 @@ cics_gds_connect: CONNECT (PROCESS | CONVID cics_name | PROCNAME cics_name | PRO
                   RETCODE cics_data_area | STATE cics_cvda | cics_handle_response)+;
 cics_gds_extract: EXTRACT (cics_gds_attributes | cics_gds_process);
 cics_gds_attributes: ATTRIBUTES (CONVID cics_name | STATE cics_cvda | CONVDATA cics_data_area | RETCODE cics_data_area | cics_handle_response)+;
-cics_gds_process: PROCESS (CONVID cics_name | cics_extract_procname | SYNCLEVEL cics_data_area |
-                  cics_extract_piplist | RETCODE cics_data_area | cics_handle_response)+;
+cics_gds_process: PROCESS (CONVID cics_name | (PROCNAME cics_data_area (PROCLENGTH cics_data_area | MAXPROCLEN cics_data_value | cics_handle_response)+) | SYNCLEVEL cics_data_area |
+                  (PIPLIST cics_ref PIPLENGTH cics_data_area) | RETCODE cics_data_area | cics_handle_response)+;
```

Comment:
I don't understand this change.

Also `cics_handle_response` is now mentioned twice.